src/components/Global/RadioButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import styles from './RadioButton.module.scss';
+
+type RadioButtonProps = {
+  id: string /*radioEasy*/;
+  name: string /*filter*/;
+  difficulty: string /*Easy>> */;
+  checked: boolean;
+  setChecked: (wasChecked: any) => void;
+};
+const getLabelColor = (difficulty: string, isChecked: boolean) => {
```

Comment:
I think you can improve this function by separating checking "checked" and checking difficulty. You can check the difficulty first then check if checked, then merge them later.